packages/cli/src/commands/items/attach-file.ts

```diff
@@ -27,42 +28,68 @@ export default class ItemsAttachFile extends MeecoCommand {
   static args = [];
 
   async run() {
-    throw new CLIError(`Command not implement`);
+    const { flags } = this.parse(this.constructor as typeof ItemsAttachFile);
+    const environment = await this.readEnvironmentFile();
+    const { auth, config } = flags;
 
-    // const { flags } = this.parse(this.constructor as typeof ItemsAttachFile);
-    // const environment = await this.readEnvironmentFile();
-    // const { auth, config } = flags;
+    try {
+      const authConfig = await this.readConfigFromFile(AuthConfig, auth);
+      const fileConfig = await this.readConfigFromFile(FileAttachmentConfig, config);
 
-    // try {
-    //   const authConfig = await this.readConfigFromFile(AuthConfig, auth);
-    //   const fileConfig = await this.readConfigFromFile(FileAttachmentConfig, config);
+      if (!authConfig) {
+        this.error('Must specify a valid auth config file');
+      }
+      if (!fileConfig) {
+        this.error('Must specify a valid file attachment config');
+      }
+      const itemService = new ItemService(environment);
+      const itemFetchResult = await itemService.get(fileConfig.itemId, authConfig);
 
-    //   if (!authConfig) {
-    //     this.error('Must specify a valid auth config file');
-    //   }
-    //   if (!fileConfig) {
-    //     this.error('Must specify a valid file attachment config');
-    //   }
+      const filePath = fileConfig.file;
+      try {
+        await readFileAsBuffer(filePath);
```

Comment:
Why read the whole file here? Maybe just check if it exists if that's all you want to do?

Reply:
this is true with make this change in the futch

Reply:
Please change from try/catch with `readFileAsBuffer` to 
`if(existsSync(filePath)) { } else { error }` and submit as a new PR